indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -233,6 +245,12 @@ public TaskStatus call()
                               command.add(String.format("-Ddruid.host=%s", childHost));
                               command.add(String.format("-Ddruid.port=%d", childPort));
 
+                              if(config.isSeparateIngestionEndPoint()) {
+                                command.add(String.format("-Ddruid.indexer.task.chathandler.service=%s", "druid/chathandler"));
```

Comment:
wait...it may not help as `serviceName` is not a path because all the  "/" are replaced to ":" and the resultant String is used to announce the service..let me double check